src/apps/explorer/components/common/Search/Search.styled.ts

```diff
@@ -1,11 +1,18 @@
-import styled from 'styled-components'
+import styled, { css, FlattenSimpleInterpolation } from 'styled-components'
 import SVG from 'react-inlinesvg'
 
 export const Wrapper = styled.form`
   display: flex;
-  width: 100%;
-  max-width: 50rem;
-  margin: 0 auto;
+  ${({ className }): FlattenSimpleInterpolation =>
```

Comment:
@alongoni [`orders/Search`](https://github.com/gnosis/gp-ui/blob/691-search-by-address/src/components/orders/Search/index.tsx#L8) and [`explorer/common/Search`](https://github.com/gnosis/gp-ui/blob/691-search-by-address/src/apps/explorer/components/common/Search/index.tsx#L8) look identical, except for a new lines of CSS, 
I've added the difference through a `className`.

[`src/components/orders/Search`](https://github.com/gnosis/gp-ui/blob/691-search-by-address/src/components/orders/Search/index.tsx#L8) would be deprecated, @alfetopito we could delete it?